starters/vue3-xstate-css/src/views/GreetView.vue

```diff
@@ -0,0 +1,78 @@
+<script setup lang="ts">
+import { useMachine } from '@xstate/vue';
+import HeaderComponent from '../components/HeaderComponent.vue';
+import { greetMachine } from '../machines/greetMachine';
+
+const props = defineProps({
+  query: String,
+});
+
+const { state } = useMachine(greetMachine(props.query || ''), {
+  devTools: true,
+});
+</script>
+
+<template>
+  <main>
+    <HeaderComponent> Fetch Data from API </HeaderComponent>
+    <section class="fetch__section">
+      <p class="fetch__section-title">Message:</p>
+      <div class="fetch__section-result" data-cy="message-result">
+        <div
+          v-if="state.value === 'loading'"
+          class="fetch__section-loader"
+        ></div>
+        <p v-if="state.value === 'failure'" class="fetch__section-message-fail">
+          {{ state.context.error }}
+        </p>
+        <p v-else class="fetch__section-message-success">
+          {{ state.context.message }}
```

Comment:
For mobile, I think reducing the font size so the text is all in one line would work better here
<img width="555" alt="Screenshot 2023-04-05 at 12 03 02 PM" src="https://user-images.githubusercontent.com/67210629/230179577-c3c42285-f208-4d78-91b8-5519a2c4a4db.png">

Reply:
What about putting "Message:" on one line and the response on another? I hesitate to make text too small on mobile, though even in one line it wouldn't have to be that much smaller I don't think. I can play with this!

Reply:
I like the idea of having message and the response on separate lines